docs/core/tutorials/libraries.md

```diff
@@ -214,7 +214,7 @@ It's important to be able to test across platforms. You can use either [xUnit](h
 
 1. Set up your solution. You can do so with the following commands:
 
-   ```bash
```

Comment:
It was indicated by @mairaw that in another article a codeblock that mixed both .NET CLI commands and other commands shouldn't be `dotnetcli` marked. If this is true, these need to revert.